nautobot/dcim/api/serializers.py

```diff
@@ -418,6 +432,20 @@ class Meta:
             "computed_fields",
         ]
         opt_in_fields = ["computed_fields"]
+        # Omit the UniqueTogetherValidator that would be automatically added to validate (manufacturer, slug). This
+        # prevents slug from being interpreted as a required field.
+        validators = [UniqueTogetherValidator(queryset=DeviceType.objects.all(), fields=("manufacturer", "model"))]
```

Comment:
Note likely desirable 2.0 breaking change: require DeviceType `slug` to be globally unique on its own, rather than only locally unique per manufacturer.

Reply:
Same as RackGroup.

Reply:
Agreed, in large device type data sets, I generate the slug from the combination of manufacturer and model.

Reply:
populate_from can take a list. This could be changed from `model` to `manufacturer` and `model`. This would need a little change for the test, and I would prefer it be separate ticket if that is alright with you both?